src/webportal/src/app/job/job-view/fabric/job-detail/components/task-role-container-list.jsx

```diff
@@ -218,40 +218,76 @@ export default class TaskRoleContainerList extends React.Component {
     );
   }
 
-  showSshInfo(id) {
+  showSshInfo(hasUserSsh, id, containerPorts, containerIp) {
     const { sshInfo } = this.context;
```

Comment:
the `hasUserSsh` is duplicate, you could get jobConfig by `const { jobConfig, sshInfo } = this.context;`

Reply:
Done.